lib/workers/global/autodiscover.ts

```diff
@@ -105,7 +105,7 @@ export function applyFilters(repos: string[], filters: string[]): string[] {
       }
       res = repos.filter(autodiscoveryPred);
     } else {
-      res = repos.filter(minimatch.filter(filter));
+      res = repos.filter(minimatch.filter(filter, { nocase: true }));
```

Comment:
This is potentially a breaking change for some

Reply:
The intention here was to actually _avoid_ a breaking change for existing users of Bitbucket server who needed to specify lowercase project keys in `autodiscoverFilter` so far. `nocase: true` broadens the scope of what is accepted, so I think existing config shouldn't be impacted by this change.

Do you suspect an issue with other platforms?

Reply:
It's probably ok, let's get another opinion though

Reply:
I think it's not breaking. I think all platforms are case insensitive on orgs anyways. I've never seen a platform where you can have orgs who only differ in case